about/glossary.rst

```diff
@@ -37,7 +56,7 @@ Glossary
       Digital Earth Australia, a platform that uses spatial data and images 
```

Comment:
DEA is a program not a platform?

Reply:
DEA Sandbox is the platform

Reply:
Is it both? Naturally DEA is a program... but is the DEA offering also a platform in a sense?